.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+# See https://circleci.com/docs/2.0/language-javascript/
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:11-browsers
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - global-deps-{{ checksum "package-lock.json" }}
+          - global-deps-
+
+      - restore_cache:
+          keys:
+          - regl-worldview-deps-{{ checksum "package-lock.json" }}
```

Comment:
Looks like this doesn't match the `checksum "packages/regl-worldview/package-lock.json"` used for this key in the `save_cache` below